Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM docker.io/gradle:jdk19-jammy AS build
+COPY --chown=gradle:gradle . /home/gradle/codecv
+WORKDIR /home/gradle/codecv
+RUN gradle clean build --no-daemon
+
+FROM docker.io/amazoncorretto:19
```

Comment:
If the project is packaged with the `jpackage` Gradle task, the bundle includes the minimum JRE needed (generated in `build/codecv`), we could use that bundle and be sure to use the same Java version, also, I guess the `FROM` could be changed to `busybox` or `alpine`. What do you think?

Reply:
That I have to check and research tomorrow. Sounds good, but I'm not so familiar with Gradle. Will research and get back to you.